README.md

```diff
@@ -1 +1,25 @@
-# opentelemetry-cpp-contrib
+# OpenTelemetry C++ Contrib
+
+This repository contains set of components extending functionality of the
+OpenTelemetry SDK. Instrumentation libraries, exporters, and other components
+can find their home here.
+
+## Contributing
+
+For information on how to contribute, consult [the contributing
+guidelines](./CONTRIBUTING.md)
+
+## Support
+
+This repository is maintained by [C++ SDK
+maintainers](https://github.com/orgs/open-telemetry/teams/cpp-maintainers)
+team and [C++ SDK
+approvers](https://github.com/orgs/open-telemetry/teams/cpp-approvers) can
+help with reviews and code approval. However, as individual components are
+developed by numerous contributors, approvers and maintainers are not expected
+to directly contribute to every component.
+
```

Comment:
Probably, we can bring more clarity ( in another PR ) on how the role of approvers and maintainers in PR review. I like the way it is done for `opentelemetry-collector-contrib` (https://github.com/open-telemetry/opentelemetry-collector-contrib):
``
News PRs will be automatically associated with the reviewers based on CODEOWNERS. PRs will be also automatically assigned to one of the maintainers or approvers for facilitation.
The facilitator is responsible for helping the PR author and reviewers to make progress or if progress cannot be made for closing the PR.
``

Reply:
Let's do this in a follow-up PR

Reply:
I think we want to follow what folks are doing here https://github.com/open-telemetry/opentelemetry-dotnet-contrib#support.
I don't think we should assume the opentelemetry/cpp-maintainers and opentelemetry/cpp-approvers are the right group for this contrib repo.

Reply:
I removed the notion of `opentelemetry/cpp-maintainers` and `opentelemetry/cpp-approvers`, and added a reference to the `CODEOWNER`S process instead.